renderer/lib/torrent-poster.js

```diff
@@ -4,12 +4,18 @@ var captureVideoFrame = require('./capture-video-frame')
 var path = require('path')
 
 function torrentPoster (torrent, cb) {
-  // First, try to use the largest video file
+  // First, try to use a poster image if available
+  var posterFile = torrent.files.filter(function (file) {
+    return file.name === 'poster.jpg'
+  })[0]
+  if (posterFile) return torrentPosterFromImage(posterFile, torrent, cb)
```

Comment:
I think we should support `torrent.gif`, `torrent.png`, and `torrent.jpg`.

Currently, if Webtorrent picks an animated gif as a torrent poster, it actually animates. This can look cool esp if people keep it subtle, but it can also become distracting.